example/spec/serial/AsyncOperationRunnerFailingLambdaSpec.js

```diff
@@ -51,13 +43,19 @@ describe('The AsyncOperation task runner executing a failing lambda function', (
         Key: payloadKey,
         Body: JSON.stringify([1, 2, 3]),
       }).promise();
-      await asyncOperationModel.create({
+
+      const asyncOperationObject = {
         id: asyncOperationId,
         taskArn: randomString(),
         description: 'Some description',
         operationType: 'ES Index',
         status: 'RUNNING',
-      });
+      };
+
+      const response = await createAsyncOperation({ prefix: config.stackName, asyncOperation: asyncOperationObject });
+      if (response.statusCode !== 200) {
```

Comment:
You won't end up needing this once all of the integration test fixes make their way into the feature branch, which probably happens before this gets merged

Reply:
[`c94c2c1` (#2294)](https://github.com/nasa/cumulus/pull/2294/commits/c94c2c131222b4f829c8a3e4e9f620dc256e0429)